civitai-downloader-v2/src/core/logging_config.py

```
@@ -0,0 +1,238 @@
+"""
+Centralized logging configuration for CivitAI Downloader v2.
+Provides file-based logging with rotation and proper formatting.
+"""
+
+import logging
+import logging.handlers
+from pathlib import Path
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)